static/js/info_overlay.js

```diff
@@ -149,6 +150,135 @@ This text won't be visible until the user clicks.
     },
 ];
 
+const search_operator_rows = [
+    {
+        operator_key: "stream",
+        operator_value: "stream",
+        effect: "Narrow to messages on stream",
+    },
+    {
+        operator_key: "topic",
+        operator_value: "topic",
+        effect: "Narrow to messages with topic",
+    },
+    {
+        operator_key: "pm-with",
+        operator_value: "email",
+        effect: "Narrow to private messages with",
+    },
+    {
+        operator_key: "group-pm-with",
+        operator_value: "email",
+        effect: "Narrow to group private messages with",
+    },
+    {
+        operator_key: "sender",
+        operator_value: "email",
+        effect: "Narrow to messages sent by",
+    },
+
+    {
+        ignore_operators: true,
+        operator_key: "sender",
+        operator_value: "me",
+        effect: "Narrow to messages sent by you.",
+    },
+    {
+        operator_key: "near",
+        operator_value: "id",
+        effect: "Center the view around message ID",
+    },
+    {
+        operator_key: "id",
+        operator_value: "id",
+        effect: "Narrow to just message ID",
+    },
+    {
+        ignore_operators: true,
+        operator_key: "streams",
+        operator_value: "public",
+        effect: "Search all public streams in the organization.",
+    },
+    {
+        ignore_operators: true,
+        operator_key: "is",
+        operator_value: "alerted",
+        effect: "Narrow to messages with alert words.",
+    },
+    {
+        ignore_operators: true,
+        operator_key: "is",
+        operator_value: "mentioned",
+        effect: "Narrow to messages that mention you.",
+    },
+    {
+        ignore_operators: true,
+        operator_key: "is",
+        operator_value: "private",
+        effect: "Narrow to private messages.",
+    },
+    {
+        ignore_operators: true,
+        operator_key: "is",
+        operator_value: "starred",
+        effect: "Narrow to starred messages.",
+    },
+    {
+        ignore_operators: true,
+        operator_key: "is",
+        operator_value: "unread",
+        effect: "Narrow to unread messages.",
+    },
+    {
+        ignore_operators: true,
+        operator_key: "has",
+        operator_value: "link",
+        effect: "Narrow to messages containing links.",
+    },
+    {
+        ignore_operators: true,
+        operator_key: "has",
+        operator_value: "image",
+        effect: "Narrow to messages containing images.",
+    },
+    {
+        ignore_operators: true,
+        operator_key: "has",
+        operator_value: "attachment",
+        effect: "Narrow to messages containing uploads.",
+    },
+    {
+        ignore_operator_styling: true,
```

Comment:
Similar as the above one, just the difference that this should be more a "custom" row. 
Like: 
![new-lower](https://user-images.githubusercontent.com/53977614/121400484-43c0d680-c975-11eb-81a5-f8ff00fc13a6.png)

Reply:
See my comments above, but I think we basically want ~all of these to use this `z-value` approach, to handle variable word order in different languages.  

So we might not need a name for this model.

Reply:
Yup, That's true.